### ISSUE TITLE
Feature Request: Shift and Control Highlight in New Episodes Availible Window

### ISSUE THREAD

**Issue by Delphik** [gpodder/gpodder#593]
It would be nice to see the ability to highlight multiple podcasts by holding control/shift and clicking in the New Episodes Available window. to keep it consistent with the main window

**Comment by auouymous** [comment]
It would be nice if clicking the name could toggle the checkbox. Shift clicking the checkbox or name to set region to the state of the starting checkbox would be nice too. But checkboxes outside the region shouldn't be changed, so supporting ctrl clicking wouldn't make sense.

**Comment by elelay** [comment]
Yes: clicking a checkbox on selected lines would apply to all lines the same effect as the checkbox you click on.
